thirdeye/thirdeye-frontend/app/styles/pods/rca.scss

```diff
@@ -150,3 +150,43 @@
 .rootcause-chart-container-fluid {
   padding: 0px;
 }
+
+.rootcause-header__major {
+  font-size: 32px;
+  font-weight: bold;
+}
+
+.rootcause-header__minor {
+
+}
+
+.rootcause-header__side {
+  text-align: right;
+}
+
+.rootcause-header-textarea {
+  resize: none;
+  border: none;
+  background: none;
+
+  &:hover {
+    background-color: white;
+  }
+}
+
+.rootcause-header-button {
+  margin: 2px 0 0 0;
+  cursor: pointer;
+  background: none;
+  padding: 2px 15px;
+  border-radius: 3px;
+  border: 1px solid $te-nav-item-color;
+  color: $te-nav-item-color;
+  letter-spacing: $te-nav-letter-spacing;
+  background-color:  $te-nav-background;
+
+  &:hover {
+    border-color: white;
+    color: white;
+  }
+}
```

Comment:
Blocking comment: Since `rootcause-header` is its own component, we should create a separate styling file for it, so it's more modular and keeps `rca.scss` shorter.

Reply:
fixed